README.md

```diff
@@ -0,0 +1,27 @@
+<h1 align="center">SM213 editor</h1>
+
+<p align="center">
+  <img alt="badge-lastcommit" src="https://img.shields.io/github/last-commit/Gabeperson/sm213_editor?style=for-the-badge">
+  <img alt="badge-openissues" src="https://img.shields.io/github/issues-raw/Gabeperson/sm213_editor?style=for-the-badge">
+  <img alt="badge-license" src="https://img.shields.io/github/license/Gabeperson/sm213_editor?style=for-the-badge">
+  <img alt="badge-contributors" src="https://img.shields.io/github/contributors/Gabeperson/sm213_editor?style=for-the-badge">
+  <img alt="badge-codesize" src="https://img.shields.io/github/languages/code-size/Gabeperson/sm213_editor?style=for-the-badge">
+</p>
+
+SM213 is a web-based code editor hosted [here](https://gabeperson.github.io/sm213_editor/). It uses SM213 parser to provide live syntactic error reporting as you code.
+
+## Features
+- **Live error reporting:** Potential syntactic errors in your command are displayed as you code so that you can catch mistakes as soon as they happen. Error messages are descritive so that you can find the error easily.
+- **Browser-based:** You don't need to install anything to run. Just go to [https://gabeperson.github.io/sm213_editor/](https://gabeperson.github.io/sm213_editor/) and start writing.
+- **Different color modes:** Currently supports light and dark mode. Provides the option to adjust the color theme to your system's default.
+
+## Using the code editor
+Using the code editor does not require installing anything. It is a website currently deployed at [https://gabeperson.github.io/sm213_editor/](https://gabeperson.github.io/sm213_editor/).
+
+For local development, just clone the repo using and then use `cargo run` to run the program. If you choose to run the program locally, Rust must be installed.
```

Comment:
You need to install "Trunk (https://github.com/trunk-rs/trunk)" and run "trunk serve --open" to run it